dev/devicelab/lib/tasks/run_tests.dart

```diff
@@ -102,28 +102,34 @@ class AndroidRunOutputTest extends RunOutputTask {
 
   @override
   TaskResult verify(List<String> stdout, List<String> stderr) {
+    final String gradleTask = release ? 'assembleRelease' : 'assembleDebug';
+    final String apk = release ? 'app-release.apk' : 'app-debug.apk';
```

Comment:
i'm assuming we never test in profile mode?

Reply:
Yup that's correct, but that might be worth adding in the future though